evaluator/opa.go

```diff
@@ -156,6 +159,7 @@ func (o *OpaEvaluator) decode(result interface{}) (RuleResult, error) {
 	}
 
 	err = decoder.Decode(result)
+	opaResult.Metadata.CreatedAt = now()
```

Comment:
Since it's only used once that could maybe be 
```suggestion
	opaResult.Metadata.CreatedAt = time.Now().UTC()
```

Reply:
The reason for this was to have the ability to overwrite the value to a predictable one in the [test](https://github.com/elastic/cloudbeat/pull/346/files#diff-eb9ab193881d8642e921206e9c7c64f27a626defd0e3cc3a2f2b8e3773e46c0cR59)